content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,42 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+Let's say we want to print the filename also in the description and so we update
+the `process.py` as:
+
+```python
+print(f'Number of lines in {sys.argv[1]}:')
+print(num_lines)
+```
+
+Now, using the `--downstream` option with `count` as a target stage, results in
+the following output:
 
 ```dvc
-$ dvc repro --downstream
-Data and pipelines are up to date.
+$ dvc repro --downstream count
+Running stage 'count' with command:
+        python process.py numbers.txt > count.txt
+Updating lock file 'dvc.lock'
 ```
 
-The reason being that the `text.txt` file is not a dependency in the last stage
-of the pipeline, used as the default target by `dvc repro`. `text.txt` is a
-dependency of the `filter` stage, which happens earlier (shown in the figure
-below), so it's skipped given the `--downstream` option.
+The change in the `text.txt` file is ignored because that file is a dependency
```

Comment:
Missed a change from my suggestion in https://github.com/iterative/dvc.org/pull/1624#discussion_r462085536
```suggestion
The change the `text.txt` is ignored because that file is a dependency
```